hardware/esp8266com/esp8266/cores/esp8266/Esp.h

```diff
@@ -78,6 +78,7 @@ class EspClass {
         void restart(void);
 
         uint16_t getVcc(void);
+        uint16_t getVdd33(void);
```

Comment:
Like i said, getVcc works differently than readvdd33. I'll go through the
appendix documentation and add info about both in the read me.

On Fri, May 15, 2015, 23:46 Ivan Grokhotkov <notifications@github.com>
wrote:

> In hardware/esp8266com/esp8266/cores/esp8266/Esp.h
> <https://github.com/esp8266/Arduino/pull/250#discussion_r30458157>:
>
> > @@ -78,6 +78,7 @@ class EspClass {
> >          void restart(void);
> >
> >          uint16_t getVcc(void);
> > +        uint16_t getVdd33(void);
>
> Why do we need another function instead of replacing getVcc implementation?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/esp8266/Arduino/pull/250/files#r30458157>.
>